packages/schematics/angular/guard/index_spec.ts

```diff
@@ -118,4 +114,46 @@ describe('Guard Schematic', () => {
     expect(fileString).not.toContain('CanLoad');
     expect(fileString).not.toContain('canLoad');
   });
+
+  it('should add correct imports based on CanLoad implementation', async () => {
+    const implementationOptions = ['CanLoad'];
+    const options = { ...defaultOptions, implements: implementationOptions };
+    const tree = await schematicRunner.runSchematicAsync('guard', options, appTree).toPromise();
+    const fileString = tree.readContent('/projects/bar/src/app/foo.guard.ts');
+    const expectedImports = `import { CanLoad, Route, UrlSegment, UrlTree } from '@angular/router';`;
+    const wrongOrderImports = `import { UrlTree, CanLoad, Route, UrlSegment } from '@angular/router';`;
```

Comment:
The test for wrong imports here is tightly is not testing anything other than that the ordering is correct, which is already testing in the `expectedImports` hence, this makes it redundant.

Reply:
Can you please remove the redundant expects? thanks

Reply:
done